src/languages/en.ts

```diff
@@ -2707,7 +2707,7 @@ const translations = {
         },
         publicDomainError: {
             title: 'Get started with Expensify Travel',
-            message: `You'll need to use your work email (e.g., name@company.com) with Expensify Travel, not your personal email (e.g., name@gmail.com).`,
+            message: `You'll need to use your work email (e.g., name@company.com) with Expensify Travel, not your personal email (e.g., name@gmail.com)`,
```

Comment:
Do we need to remove the period here? If so, then `es.ts` should also be updated for `publicDomainError.message`.